contrib/examples/cluster-api-controllers-template.yaml

```diff
@@ -41,10 +41,8 @@ objects:
         serviceAccountName: cluster-api-controller-manager
         containers:
         - name: machine-controller
-          image: cluster-operator:latest
+          image: " "
```

Comment:
What's the trick here, and if there is one could you add a comment on it. I'm confused where the fake machine controller imagestream is getting created, is it just implicit because we pushed an image?

Reply:
Yes, pushing an image to the integrated registry will create an imagestream